app/src/androidTest/java/org/mozilla/fenix/syncintegration/gradlewbuild.py

```diff
@@ -21,20 +21,23 @@ def test(self, identifier):
 
         # Change path accordingly to go to root folder to run gradlew
         os.chdir('../../../../../../../..')
-        args = './gradlew ' + 'app:connectedGeckoNightlyDebugAndroidTest -Pandroid.testInstrumentationRunnerArguments.class=org.mozilla.fenix.syncintegration.SyncIntegrationTest#{}'.format(identifier)
+        cmd = './gradlew ' + 'app:connectedGeckoNightlyDebugAndroidTest -Pandroid.testInstrumentationRunnerArguments.class=org.mozilla.fenix.syncintegration.SyncIntegrationTest#{}'.format(identifier)
 
-        self.logger.info('Running: {}'.format(' '.join(args)))
+        self.logger.info('Running cmd: {}'.format(cmd))
```

Comment:
This is a single string, so doing `' '.join` was causing it to be logged with a space between each letter, like `Running . / g r a d l e w ...`.